migtests/scripts/run-test-export-data.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $# -ne 1 ]
+then
+	echo "Usage: $0 TEST_NAME"
+	exit 1
+fi
+
+set -x
+
+export TEST_NAME=$1
+
+export REPO_ROOT="${PWD}"
+export SCRIPTS="${REPO_ROOT}/migtests/scripts"
+export TESTS_DIR="${REPO_ROOT}/migtests/tests"
+export TEST_DIR="${TESTS_DIR}/${TEST_NAME}"
+export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/export-dir"}
+
+export PYTHONPATH="${REPO_ROOT}/migtests/lib"
+
+# Order of env.sh import matters.
+source ${TEST_DIR}/env.sh
+source ${SCRIPTS}/${SOURCE_DB_TYPE}/env.sh
+source ${SCRIPTS}/yugabytedb/env.sh
+
+source ${SCRIPTS}/functions.sh
+
+main() {
+	echo "Deleting the parent export-dir present in the test directory"
+	rm -rf ${EXPORT_DIR}	
+	echo "Creating export-dir in the parent test directory"
+	mkdir -p ${EXPORT_DIR}
+	echo "Assigning permissions to the export-dir to execute init-db, cleanup-db scripts"
+	chmod +x ${TEST_DIR}/init-db ${TEST_DIR}/cleanup-db
+
+	step "START: ${TEST_NAME}"
+	print_env
+
+	pushd ${TEST_DIR}
+
+	step "Initialise source database."
+	./init-db
+
+	step "Grant source database user permissions"
+	grant_permissions ${SOURCE_DB_NAME} ${SOURCE_DB_TYPE} ${SOURCE_DB_SCHEMA}
+
+	step "Check the Voyager version installed"
+	yb-voyager version
+
+	step "Export data."
+	# false if exit code of export_data is non-zero
+	export_data || { 
+		cat_log_file "yb-voyager-export-data.log"
+		cat_log_file "debezium-source_db_exporter.log"
+		exit 1
+	}
+
+	ls -l ${EXPORT_DIR}/data
+	cat ${EXPORT_DIR}/data/export_status.json || echo "No export_status.json found."
+	cat ${EXPORT_DIR}/metainfo/dataFileDescriptor.json
+
+	step "Fix data."
+	if [ -x "${TEST_DIR}/fix-data" ]
+	then
+		"${TEST_DIR}/fix-data" ${EXPORT_DIR}
+	fi
+	
+	step "End Migration: clearing metainfo about state of migration from everywhere."
+	end_migration --yes
+
+	step "Clean up"
+	./cleanup-db
+	rm -rf "${EXPORT_DIR}/*"
+	run_ysql yugabyte "DROP DATABASE IF EXISTS ${TARGET_DB_NAME};"
+}
+
```

Comment:
we can run the whole migration as well, and in that case, we can use `run-test.sh`??

Reply:
I purposely went with only export-data. Reason being - we already have a test that tests entire migration of case sensitive table names. 
In this case, the bug was only in export-data, so it would be redundant to test out the whole migration.